src/components/Editor/Runners/HtmlRunner/HtmlRunner.jsx

```diff
@@ -83,6 +92,18 @@ function HtmlRunner() {
     return URL.createObjectURL(blob);
   };
 
+  const getFilename = (iframe) => {
+    let filename;
+    if (iframe) {
+      filename = iframe.querySelectorAll("meta[filename]")[0]
+        ? iframe.querySelectorAll("meta[filename]")[0].getAttribute("filename")
+        : externalLink;
+    } else {
+      filename = externalLink;
+    }
+    return filename;
+  };
```

Comment:
Would prob be better to return from the `if` then ditch the `else` statement I think

Reply:
I don't mind - I tend to find things that return early a bit confusing, but that's just personal preference